src/Config/DatabaseObject.cs

```diff
@@ -153,6 +167,23 @@ public bool IsAnyColumnNullable(List<string> columnsToCheck)
                                  .Where(isNullable => isNullable == true)
                                  .Any();
         }
+
+        /// <summary>
+        /// Method to get the DbType for:
+        /// 1. column for table/view,
+        /// 2. parameter for stored procedure.
+        /// </summary>
+        /// <param name="paramName">The parameter whose DbType is to be determined.</param>
+        /// <returns></returns>
+        public virtual DbType? GetDbTypeForParam(string paramName)
+        {
+            if (Columns.TryGetValue(paramName, out ColumnDefinition? columnDefinition))
```

Comment:
>Columns.TryGetValue(paramName [](http://example.com/codeflow?start=16&length=29)

Since `paramName` is used as a lookup key in the columns collection, can you add a comment to instruct callers to use the "backing" column name? If the columns have mapped aliases, what is the behavior here? Does the caller use "trygetbackingcolumn()" ?

Reply:
Added to the summary of the base method. Yes, the caller uses tryGetBackingColumnName().